### PR TITLE
Adjusted combined given-when steps in WebUIAdminSharingSettings context

### PR DESCRIPTION
**Pull request by kiranparajuli589**
## Description
Combined `given-when` steps splitted into seperate functions and `check-success` assertion added on **Given** steps.

## Related Issue
- Part of https://github.com/owncloud/QA/issues/635

## How Has This Been Tested?

- CI
- :robot:

## Checklist:

- [x] Code changes

### CODE REVIEW

File: tests/acceptance/features/bootstrap/WebUIAdminSharingSettingsContext.php [hunk 1]
```diff
@@ -315,6 +314,43 @@ public function theAdministratorEnablesExcludeGroupsFromSharingUsingTheWebui() {
...
+			$this->featureContext->getBaseUrl(),
+			$this->featureContext->getOcPath()
+		);
+		$status = \trim($response['stdOut']);
+		Assert::assertEquals(
+			"yes",
+			$status
+		);
+	}
+
```
Comment (HariBhandari07) [inline tests/acceptance/features/bootstrap/WebUIAdminSharingSettingsContext.php]: I think this belongs in OccContext

File: tests/acceptance/features/bootstrap/WebUIAdminSharingSettingsContext.php [hunk 2]
```diff
@@ -406,6 +441,50 @@ public function theAdministratorAddsGroupToTheExcludeGroupFromSharingList($group
...
+			$this->featureContext->getOcPath()
+		);
+		$response = SetupHelper::runOcc(
+			[
+				'config:app:get',
+				'core',
+				'shareapi_exclude_groups_list'
+			],
+			$this->featureContext->getAdminUsername(),
+			$this->featureContext->getAdminPassword(),
```
Comment (HariBhandari07) [inline tests/acceptance/features/bootstrap/WebUIAdminSharingSettingsContext.php]: same as above

Comment (kiranparajuli589) [reply tests/acceptance/features/bootstrap/WebUIAdminSharingSettingsContext.php]: done!

File: tests/acceptance/features/bootstrap/WebUIAdminSharingSettingsContext.php (hunk 1 above)
Comment (kiranparajuli589) [reply tests/acceptance/features/bootstrap/WebUIAdminSharingSettingsContext.php]: done!

File: tests/acceptance/features/bootstrap/OccContext.php [hunk 3]
```diff
@@ -1521,6 +1521,87 @@ public function enableExternalStorageUsingOccAsAdmin() {
 		);
 	}
 
+	/**
+	 * @Given the administrator has added group :group to the exclude group
```
Comment (phil-davis) [inline tests/acceptance/features/bootstrap/OccContext.php]:
```suggestion
	 * @Given the administrator has added group :group to the exclude group from sharing list
```

File: tests/acceptance/features/bootstrap/OccContext.php [hunk 4]
```diff
@@ -1521,6 +1521,87 @@ public function enableExternalStorageUsingOccAsAdmin() {
...
+	 *
+	 * @param string $groups
+	 * multiple groups can be passed as comma separated string
+	 *
+	 * @return void
+	 * @throws Exception
+	 */
+	public function theAdministratorHasAddedGroupToTheExcludeGroup($groups) {
```
Comment (phil-davis) [inline tests/acceptance/features/bootstrap/OccContext.php]:
```suggestion
	public function theAdministratorHasAddedGroupToTheExcludeGroupFromSharingList($groups) {
```

File: tests/acceptance/features/bootstrap/OccContext.php (hunk 3 above)
Comment (kiranparajuli589) [reply tests/acceptance/features/bootstrap/OccContext.php]: done!

File: tests/acceptance/features/bootstrap/OccContext.php (hunk 4 above)
Comment (kiranparajuli589) [reply tests/acceptance/features/bootstrap/OccContext.php]: done!

**Review by phil-davis — APPROVED** [review]

**Pull request merged**